setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 
-VERSION = '1.40'
+VERSION = '1.41'
```

Comment:
Bumping new releases is up to project owners/maintainers. Also, it doesn't really belongs to any specific fix/feature one tries to solve/implement.

I believe you changed it in order to have it upgraded by `pip install`. I'm not sure about your experience with Python development, but if you allow me to give you what may be a tip, you can use `pip install -U -e .` from the root of the project to have it installed in *editable mode*:

```
$ pip help install
[snip]
Install Options:
  -e, --editable <path/url>   Install a project in editable mode (i.e. setuptools "develop mode")
                              from a local project path or a VCS url.
[snip]
  -U, --upgrade               Upgrade all specified packages to the newest available version. This
                              process is recursive regardless of whether a dependency is already
                              satisfied.
[snip]
```

Also, allow me to strongly recommend you using [Virtualenv](http://virtualenv.readthedocs.org/en/latest/), if you're not already doing so